features/school_form_submission.feature

```diff
@@ -5,9 +5,9 @@ Feature: view schools
 
```

Comment:
Nice tests! Is it possible to come up with a test that tests undesired user behaviour? For example, something that will result into an error raised, or other behavior that will trigger an unhappy path.